system-tests/complex-config/.codeclimate.yml

```diff
@@ -4,4 +4,5 @@ plugins:
     enabled: true
     config:
       config: "eslintrc.json"
-    include_paths: [ "/usr/src/app/integration/extends_airbnb/index.js" ]
+    include_paths: 
+      - "/usr/src/app/integration/extends_airbnb/index.js"
```

Comment:
does this really works now ??, I see some related changes at `inclusionBasedFileListBuilder` on `lib/eslint.js` but not seeing how is it that now we support this other format.